doc/Extensions/Applications.md

```diff
@@ -258,7 +258,7 @@ options {
 3: Restart your bind9/named after changing the configuration.
 
 4: Verify that everything works by executing `rndc stats && cat
-/var/run/named/stats`. In case you get a `Permission Denied` error,
+/var/cache/bind/stats`. In case you get a `Permission Denied` error,
```

Comment:
So, you changed a path that is correct for some OS/distros to a path that is correct for other OS/distros?

Reply:
Step 1 says to create the file in /var/cache/bind/stats and set its permissions, so I just changed point 4 to match it.

Reply:
Sounds good, reading snippets on PRs doesn't always give the full context.